includes/widgets/video.php

```diff
@@ -314,21 +314,14 @@ protected function register_controls() {
 			'autoplay',
 			[
 				'label' => esc_html__( 'Autoplay', 'elementor' ),
+				'description' => sprintf(
+					esc_html__( 'Note: Autoplay is affected by %s', 'elementor' ),
+					'<a href="https://developers.google.com/web/updates/2017/09/autoplay-policy-changes" target="_blank">' .
+					esc_html__( 'Google’s Autoplay policy', 'elementor' ) . '</a>' .
+					esc_html__( ' on Chrome browsers.', 'elementor' )
+				),
 				'type' => Controls_Manager::SWITCHER,
 				'frontend_available' => true,
-				'conditions' => [
-					'relation' => 'or',
-					'terms' => [
-						[
-							'name' => 'show_image_overlay',
-							'value' => '',
-						],
-						[
-							'name' => 'image_overlay[url]',
-							'value' => '',
-						],
-					],
-				],
```

Comment:
why did you removed the conditions?

Reply:
I removed these conditions, because it is no longer needed.

Reply:
I understand that from the commit, but why they are not needed?

Reply:
They are no longer needed, if someone wants to enable/disable autoplay on lightbox it will not work.

Reply:
You now cannot disable or enable autoplay on a overlay image/lightbox.